### DISCUSSION TITLE
Eradicator Deathdroid

### DISCUSSION THREAD

**Discussion by TomekPL-1** [secretcowlevel/DoomtrooperPublic#3] [Card Balance]
Eradicator Deathdroid (CYBERTRONIC) [R]
NOEQUIP, NOCOVER, POWERIMMUNITY
May never be in Cover. At the beginning of your turn AUGMENT 1 time (flamethrower: +2 to F/S,
Saw: ADDS SLAYER, ChainGun: +4 to S, Auto repair system: +2A)
F: 4 S: 7 A: 6 V: 8

**Comment by Kite80** [comment]
Saw augment should give it slayer only in F combat. Btw do you mean it gains 1 augment each turn with no limits? Or shall each augment be taken once?

**Comment by TomekPL-1** [comment]
I was thinking to Augment every turn without any limits. With lowered stats he need two more turns after muster to reach stats of old version of his card. 3 turns is a lot of time for opponent to get rid of him.

We already have such mechanic in play: Yojimbo and Club Arkadin

Types of Augments are only examples

**Reply by 21-ric** [reply]
I'm not a fan of AUGMENT.  :no_mouth:
But I'll try to help on this card.

Eradicator Deathdroid (CYBERTRONIC) [R]
NOEQUIP, NOCOVER, POWERIMMUNITY
May never be in Cover. At the beginning of your turn AUGMENT 1 time (classic AUGMENT). 
F: 3 S: 6 A: 5 V: 7

Work with "mirrorman".

**Reply by TomekPL-1** [reply]
In my opinion stats F: 3 S: 6 A: 5 V: 7 are too low for him.
Like Justin sayed before- Eradicator should have different augments, more UNIQUE. 3 turns to reach his full potential and more than 3 turns to make him stronger than old version is fair price.

If he has his own Unique augments Mirrorman should not benefit from this, but if he has normal Augments then I totally agree with 21-ric that it should work with Mirrorman (anotgher Cybertronic synergy)

**Comment by 21-ric** [comment]
We need to be careful with AUGMENT every turn .
Especially for AUGMENT A+2. 
Higher stats + cards to avoid fight + cards for heal could transform "eradicator deathdroid' as an indestructible warrior.

**Reply by TomekPL-1** [reply]
Oh please. 
Eradicator Deathdroid was Cybertronic strongest warrior.

With this ability and stats:

Eradicator Deathdroid (CYBERTRONIC) [R]
NOEQUIP, NOCOVER, POWERIMMUNITY
May never be in Cover. At the beginning of your turn AUGMENT 1 time (flamethrower: +2 to F/S,
Saw: ADDS SLAYER, ChainGun: +4 to S, Auto repair system: +2A)
F: 4 S: 7 A: 6 V: 8

And widhout other cards he can wound and also be wounded by Bio Gigant 2 turns after deploying. And 4 turns to wound Bio Gigant by S without beying wounded. And by F he beats him after more than 10 turns...

This is fair price in my opinion for this kind of ability.

And if not we are here to nerf him later

**Comment by 21-ric** [comment]
Mmmm 
What do you think if we give your suggestion to"Scrap Heap"?

Scrap Heap (CYBERTRONIC) [R]
INFLUENCE
At the end of your turn choose one of your CYBERTRONIC WARRIOR : AUGMENT 1.

**Reply by framerate** [reply]
I think this is too strong, honestly. Especially with Mirrorman

**Comment by framerate** [comment]
Yeah just to reply to this from a high level -

He is set to get 3 unique augments (based on lore) that are only for him. If you look up the lore, we can assign arbitrary names. But what the 3 augments are... thats the delay...

**Reply by Sol-au-fein** [reply]
Titan Megablaster - upgraded SSW4200P heavy machine gun

Gigadeath chainripper - chainsaw

SSW6000 - Rocket Launcher - anti vehicle

AR3002 - modified assault rifle

Magmaschorcher - flamethrower

SA-SG80 anti-personnel system - shotgun

**Comment by TomekPL-1** [comment]
OK. New direction -- totally crazy :o)  Two versions of Eradicator Deathdroid on one card. When played player choose which one he want to have:

![Eradictor](https://user-images.githubusercontent.com/83038538/116215894-5fb34480-a748-11eb-863d-666e2b038fd5.jpg)

Eradicator Deathdroid (CYBERTRONIC) [R]
NOEQUIP, NOCOVER, POWERIMMUNITY 
May never be in Cover. 
F: 6 S: 9 A: 8 V: 9


Eradicator Deathdroid Construction Kit (CYBERTRONIC) [R]
NOEQUIP, NOCOVER, POWERIMMUNITY
May never be in Cover. At the end of your turn AUGMENT 1 time. If there is VAC Engineer in your squad AUGMENT 1 time at the beginning of your turn. (SA-SG80 anti-personnel system: +2 to F/S, Titan Megablaster: +4 to S, Auto repair system: +2A, Overclock CPU: ADDS FIRST STRIKE, Magmaschorcher: ADDS SLAYER, ) Every time Eradicator AUGMENT he gain +1 V
F: 4 S: 3 A: 4 V: 5

Eradicator construction kit AUGMENT always 1 time per turn (on the end of the turn), or Two times per turn when there is VAC engineer in play(at the beginning 1 time and at the end of turn 1 time).

Eradicator Deathdroid AUGMENTS are UNIQUE and cannot affect other cards/warriors like Mirrorman.

What do you think??

**Comment by 21-ric** [comment]
What will look like our "eradicator deathdroid"?

We have many clues for build a good card.

@TomekPL-1 Eradicator Deathdroid Construction Kit is very smart but I don't think this idea will be retained.

2 augments among several unique options. 

@Kite80 I like the idea to have POWERIMMUNITY in one option.
It will be very cool if "mirrorman" gain augments from "eradicator deathroid". 
:arrow_right: "mirrorman" gain +1V each time he receives an AUGMENT.

**Comment by 21-ric** [comment]
Classic idea :
Eradicator Deathdroid (CYBERTRONIC) [R]
NOEQUIP, NOCOVER. 
May never be in Cover.
AUGMENTS : 2
F: 4 S: 8 A: 7 V: 9
Unique augment 1 : POWERIMMUNITY
Unique augment 2 : S+5
Unique augment 3 : A+3

Could works with "mirrorman".
➡️ "mirrorman" gain +1V each time he receives an AUGMENT.
➡️ "mirrorman" may only receive unique augment once.

**Comment by TomekPL-1** [comment]
Augments depends from VAC engineers

![Eradictor-1](https://user-images.githubusercontent.com/83038538/118521324-35e3b100-b73b-11eb-8f65-4b6ee1d4351f.jpg)


Eradicator Deathdroid (CYBERTRONIC) [R]
NOEQUIP, NOCOVER, POWERIMMUNITY
May never be in Cover.
F: 6 S: 9 A: 8 V: 9

Eradicator Deathdroid Construction Kit (CYBERTRONIC) [R]
NOEQUIP, NOCOVER, POWERIMMUNITY
May never be in Cover. For every VAC Engineer in your squad AUGMENT 1 time at the beginning of your turn. (SA-SG80 anti-personnel system: +2 to F/S, Titan Megablaster: +4 to S, Auto repair system: +2A, Overclock CPU: ADDS FIRST STRIKE, Magmaschorcher: ADDS SLAYER, ) Every time Eradicator AUGMENT he gain +1 V
F: 4 S: 3 A: 4 V: 5

Eradicator Deathdroid AUGMENTS are UNIQUE and cannot affect other cards/warriors like Mirrorman.

**Reply by framerate** [reply]
I don't love these "when ONE SPECIFIC card is in your squad" rules though, requires players to have them both to work (Shogun/Scalper is another one).

You like that? Seems like it'll cause problems!

**Reply by TomekPL-1** [reply]
If you already lost all your Vac engineers you can muster orginal version of Eradicator ( without Augments). Later in the game if there is more engineers warrior in game it can be changed  from VAC engineer to engineer key Word

And this solution gives opponent way to stop Eradictor augmentation by killing all engineers

**Comment by david-roxygen** [comment]
In my opinion Eradicator Deathdroid needs a little boost. I suggest to add 'when played AUGMENT 1 time'  (considering also that can't be equipped with attachments).

![Picture9](https://user-images.githubusercontent.com/88280663/130096656-45c4598b-4e5c-43a6-b203-9ab2f83bf29e.jpg)

**Reply by Sol-au-fein** [reply]
Lame!

**Reply by david-roxygen** [reply]
Do you mean that we should change based on lore?! XD

![image](https://user-images.githubusercontent.com/88280663/130108284-408ecea8-8f7c-4084-ab80-38e8f569d255.png)

![image](https://user-images.githubusercontent.com/88280663/130106116-372732e5-692a-4ad5-b904-61b3ce5e6407.png)

![image](https://user-images.githubusercontent.com/88280663/130106587-4742033e-7dda-4c4b-9ef7-df9a91ba039b.png)

**Reply by david-roxygen** [reply]
It is hard to suggest a balanced cards with this background XD. 

could be

Eradicator Deathdroid (CYBERTRONIC) [UR]
NOEQUIP, NOCOVER, POWERIMMUNITY
F:6 S:9 A:8 V:12

[! wrong text ]At the beginning of your turn AUGMENT 1 time 
[! right text] When played AUGMENT 1 time


-Gigadeath chainripper: +4F and slayer
-Titan Megablaster (i.e. upgraded SSW4200P heavy machine gun): +4S and slayer
-Self-repair carapace: +4 A and 1 action to heal 1 wound

**Reply by Sol-au-fein** [reply]
Not balanced

**Reply by Sol-au-fein** [reply]
Do you really want him to get +4 and slayer every turn?

**Reply by david-roxygen** [reply]
It should be evaluated togheter to alakay cardinal Dominique and biogigant.
In a hypothetical combat vs alakay, Eradicator could damage alakay only with +4S bonus.
In a hypothetical combat vs Cardinal Dominique, Eradicator could damage Cardinal with +4S and +4F bonus.
Alakay and Dominique can damage eradicator without big problems.
Only with +4A eradicator could survive.
Biogigant is still stronger than eradicator, but eradicator is more versatile 

Maybe slayer could be forget

**Reply by Sol-au-fein** [reply]
I will ask you one more time. Do you really want him to get +4 augment every turn? So in second turn he is +8, third +12, 4th +16?

**Reply by david-roxygen** [reply]
No, only one time. Ops, I copy past a previous version XD

**Reply by david-roxygen** [reply]
Eradicator Deathdroid (CYBERTRONIC) [UR]
NOEQUIP, NOCOVER, POWERIMMUNITY
F:6 S:9 A:8 V:12

When played AUGMENT 1 time

-Gigadeath chainripper: +4F and slayer in fight combat
-Titan Megablaster (i.e. upgraded SSW4200P heavy machine gun): +4S and slayer in shoot combat
-Self-repair carapace: +4 A and 1 action to heal 1 wound

**Reply by TomekPL-1** [reply]
>
Sol-au-fein
Lame!

Polite as always XDXDXD

**Comment by Sol-au-fein** [comment]
F: 4 S: 4 A: 10 V: 10
NOEQUIP NOCOVER
Select 2 Armaments.

Well, i designed Eradicator for only two armanents out of 6. I think if you will have 3 picks, it would lead to only one OP combination every time.

Titan Megablaster: S +8
`This upgraded SSW4200P is so heavy that no human can possibly carry it. It can only be fitted onto reinforced chassis of an Eradicator Deathdroid. It has burst fire capability.`

Gigadeath Chainripper: F +4. Gain SLAYER in F combat. 
`Only an Eridicator's internal generator can power this super-powerful Chainsaw-based weapon. It can be used to make sweep attacks.`

Magmascorcher: F +4. Gain BURN in F combat. 
`Magmascorcher is flamethrower especially designed for the Eradicators. The double barrel ensures high reliability.  When one barrel reaches a critical temperature, the weapon automatically shifts over to the other, while cooling the first.`

SSW6000 Rocket Launcher: 1 TACTICAL ACTION: Deal 3* damage to target vehicle.
(* Or whatever damage will be enough for destroying light vehicle)
`The squad weapon 6000 is Cybertronic's only entry in autocannon class. It is a standard weapon of long ranged Eradicators and it is also used in several vehicles.`

Self-repair System: 3 hit points instead of 2. 
(One way or another, it could work same as lifeguard armor. Alternatively convert it into crewless vehicle, in which case it would restore durability)
`Within the massive armored carapace of the Eradicator are many self-repair mechanisms. These help keep the unit moving even when it has been shot to pieces. Some claim the only way to take out the Eradicator is to overload these mechanisms. It is true.`

Enhanced Microchip: POWERIMMUNITY

**Comment by TomekPL-1** [comment]
![118521324-35e3b100-b73b-11eb-8f65-4b6ee1d4351f](https://user-images.githubusercontent.com/83038538/143905561-88dc1687-8e84-4b72-92dd-bc259d3b7243.jpg)

Two version of Eradicator in one card - when playing owner chose which one enter SQUAD

Eradicator Deathdroid 
CYBERTRONIC) [R]
NOEQUIP, NOCOVER, POWERIMMUNITY
May never be in Cover.
F: 6 S: 9 A: 8 V: 9

Eradicator Deathdroid Construction Kit (CYBERTRONIC) [R]
NOEQUIP, NOCOVER, POWERIMMUNITY
May never be in Cover. Enter Squad with 5 counters. At the end of your turn AUGMENT 1 time. For every VAC Engineer in your squad AUGMENT 1 time at the beginning of your turn. (SA-SG80 anti-personnel system: +2 to F/S, Titan Megablaster: +4 to S, Auto repair system: +2A, Overclock CPU: ADDS FIRST STRIKE, Magmaschorcher: ADDS SLAYER, ) Every time Eradicator AUGMENT remove 1 counter and he gain +1 V
F: 4 S: 3 A: 4 V: 5

Eradicator Deathdroid AUGMENTS are UNIQUE and cannot affect other cards/warriors like Mirrorman.
